CHANGELOG.md

```diff
@@ -103,6 +103,8 @@
   [#9877](https://github.com/Kong/kong/pull/9877)
 - **JWT**: Deny requests that have different tokens in the jwt token search locations. Thanks Jackson 'Che-Chun' Kuo from Latacora for reporting this issue.
   [#9946](https://github.com/Kong/kong/pull/9946)
+- **Datadog**: Fix a bug that the batch queue in datadog can't get the right result when processing batch entries produced by datadog.
```

Comment:
Let's make this text match what's in the StatsD PR

Reply:
done